tests/unit/services/wiki-urls-test.js

```diff
@@ -341,4 +341,93 @@ module('Unit | Service | wiki-urls', (hooks) => {
 			});
 		});
 	});
+
+	test('test getEncodedTitleFromURL', (assert) => {
```

Comment:
I'd add another test that uses `wikiUrls.set('langPath', '/zh')` before calling `getEncodedTitleFromURL`